source/community/reactnative/ios/.xcode.env.local

```diff
@@ -0,0 +1,2 @@
+export NODE_BINARY=/Users/srisaimohansuri/.nvm/versions/node/v18.19.0/bin/node
```

Comment:
will this go to gitignore, or kept for sample purpose

Reply:
+1,
Shall we add this to gitignore and add a sample file for the entries alone? say, `.xcode.env.local.example`